ansible/roles/operator-pipeline/templates/openshift/tasks/create-container-image.yml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-container-image
+spec:
+  params:
```

Comment:
There seems to be a bunch of unused params here, like `cert_project_id` and `vendor_label`. If they're not needed anymore please remove them.